Build error: react-flight@1.0.0 build:umd: `NODE_ENV=development webpack`

Wanted to play around `react-flight` but wasn't been able to run `yarn`. Tested it with both `yarn` / `yarn start` and `npm install` / `npm start`

___

📝 **Log**

```
edvins.antonovs@NEMO MINGW64 /d/repos
$ git clone https://github.com/jondot/react-flight
Cloning into 'react-flight'...
remote: Counting objects: 106, done.
remote: Compressing objects: 100% (73/73), done.
remote: Total 106 (delta 22), reused 106 (delta 22), pack-reused 0
Receiving objects: 100% (106/106), 1.54 MiB | 886.00 KiB/s, done.
Resolving deltas: 100% (22/22), done.

edvins.antonovs@NEMO MINGW64 /d/repos
$ cd react-flight/

edvins.antonovs@NEMO MINGW64 /d/repos/react-flight (master)
$ ls
core/  dom/       media/        README.md  webpack.config.js
dist/  examples/  package.json  src/       yarn.lock

edvins.antonovs@NEMO MINGW64 /d/repos/react-flight (master)
$ yarn
yarn install v0.23.2
[1/4] Resolving packages...
[2/4] Fetching packages...
warning fsevents@1.0.15: The platform "win32" is incompatible with this module.
info "fsevents@1.0.15" is an optional dependency and failed compatibility check. Excluding it
 from installation.
[3/4] Linking dependencies...
[4/4] Building fresh packages...
warning Your current version of Yarn is out of date. The latest version is "0.24.6" while you
're on "0.23.2".
info To upgrade, download the latest installer at "https://yarnpkg.com/latest.msi".
$ npm run build

> react-flight@1.0.0 build D:\repos\react-flight
> npm run build:lib && npm run build:umd && npm run build:umd:min


> react-flight@1.0.0 build:lib D:\repos\react-flight
> babel src --out-dir . && rm -rf __tests__

src\core\director.js -> core\director.js
src\core\flight.js -> core\flight.js
src\core\velocity-composer.js -> core\velocity-composer.js
src\dom\controls.js -> dom\controls.js
src\dom\index.js -> dom\index.js
src\dom\ui.js -> dom\ui.js
src\__tests__\index.spec.js -> __tests__\index.spec.js

> react-flight@1.0.0 build:umd D:\repos\react-flight
> NODE_ENV=development webpack

'NODE_ENV' is not recognized as an internal or external command,
operable program or batch file.

npm ERR! Windows_NT 10.0.15063
npm ERR! argv "C:\\Program Files\\nodejs\\node.exe" "C:\\Program Files\\nodejs\\node_modules\
\npm\\bin\\npm-cli.js" "run" "build:umd"
npm ERR! node v6.10.2
npm ERR! npm  v3.10.10
npm ERR! code ELIFECYCLE
npm ERR! react-flight@1.0.0 build:umd: `NODE_ENV=development webpack`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the react-flight@1.0.0 build:umd script 'NODE_ENV=development webpack'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the react-flight package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     NODE_ENV=development webpack
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs react-flight
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls react-flight
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     D:\repos\react-flight\npm-debug.log

npm ERR! Windows_NT 10.0.15063
npm ERR! argv "C:\\Program Files\\nodejs\\node.exe" "C:\\Program Files\\nodejs\\node_modules\
\npm\\bin\\npm-cli.js" "run" "build"
npm ERR! node v6.10.2
npm ERR! npm  v3.10.10
npm ERR! code ELIFECYCLE
npm ERR! react-flight@1.0.0 build: `npm run build:lib && npm run build:umd && npm run build:u
md:min`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the react-flight@1.0.0 build script 'npm run build:lib && npm run build:um
d && npm run build:umd:min'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the react-flight package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     npm run build:lib && npm run build:umd && npm run build:umd:min
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs react-flight
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls react-flight
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     D:\repos\react-flight\npm-debug.log
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/install for documentation about this command.
```

--- a/package.json
+++ b/package.json
@@ -5,8 +5,8 @@
   "license": "MIT",
   "scripts": {
     "build:lib": "babel src --out-dir . && rm -rf __tests__",
-    "build:umd:min": "NODE_ENV=production webpack --display-modules",
-    "build:umd": "NODE_ENV=development webpack --display-modules",
+    "build:umd:min": "cross-env NODE_ENV=production webpack --display-modules",
+    "build:umd": "cross-env NODE_ENV=development webpack --display-modules",
     "build": "yarn build:lib && yarn build:umd && yarn build:umd:min",
     "release": "yarn juild && yarn publish",
     "test": "jest",
@@ -30,6 +30,7 @@
     "babel-preset-latest": "^6.24.1",
     "babel-preset-react": "^6.24.1",
     "babel-preset-stage-0": "^6.16.0",
+    "cross-env": "~5.0.1",
     "enzyme": "^2.8.2",
     "jest": "^18.0.0",
     "react": "^15.5.4",

